service/spi/src/main/java/org/kaazing/gateway/service/collections/MemoryDistributedObject.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.service.collections;
+
+import com.hazelcast.core.DistributedObject;
+
+
+public class MemoryDistributedObject implements DistributedObject {
```

Comment:
The code of the topic implementation was less polluted. But I can move these methods in MemoryTopic.